src/popupwin.c

```diff
@@ -3823,11 +3823,13 @@ update_popups(void (*win_update)(win_T *wp))
 	if (wp->w_popup_title != NULL)
 	{
 	    char_u  *title_text;
+	    int      title_byte_len;
 
-	    title_len = (int)STRLEN(wp->w_popup_title);
-	    title_text = alloc(title_len + 1);
+	    title_len = (int)MB_CHARLEN(wp->w_popup_title);
+	    title_byte_len = (int)STRLEN(wp->w_popup_title);
+	    title_text = alloc(title_byte_len + 1);
 	    trunc_string(wp->w_popup_title, title_text,
```

Comment:
`trunc_string()` might truncate the title, so I think we should set `title_len = (int)MB_CHARLEN(title_text);` after the `trunc_string()` call.

Reply:
Yes suspicious. All test are successful. Why? Is the exact value not relevant or is a test missing?
I‘ll check tonight.

Reply:
The code handling that is directly below:
```c
	    if (title_len > total_width - 2)
		title_len = total_width - 2;
```